vscode_extension/src/test/suite/aacExecutableWrapper.test.ts

```diff
@@ -1,24 +1,2 @@
-import * as assert from "assert";
-
-import * as aacWrapper from "../../aacExecutableWrapper";
-import * as configuration from "../../configuration";
-
 suite("AaC Executable Wrapper Test Suite", () => {
```

Comment:
We'll need to add some more tests pretty soon.

Reply:
Ugh, yeah. I think the extension is slim enough where it's not that big of a deal, but I would certainly feel better if we had the selenium test runner working. :/